### PR TITLE
other New Suru Icons

### PR DESCRIPTION
**Pull request by mymike00**
sorry to push this changes after you merged the other pull request, but I forgot to do it before...
mainly changes are:

- new download/stop download icons for media items (documents, audio messages, ...)
- other icons in the ProfilePage (block user, username)

I also started adding the self-chat option, but it isn't finished so I commented the lines for it and it's invisible for the user

### CODE REVIEW

**Comment by nfsprodriver** [comment]
Looks good so far. Merge?

**Comment by Flohack74** [comment]
Can you make me a screenshot just to see it before ;)

**Comment by mymike00** [comment]
Ok, wait only a moment...

File: telegram/app/qml/AccountMessageMedia.qml [hunk 1]
```diff
@@ -128,7 +128,7 @@ Item {
         id: file_handler
         telegram: telegramObject
         target: message
-        defaultThumbnail: "qrc:/qml/files/document.png"
+        defaultThumbnail: "qrc:/qml/files/document.png"
```
Comment (mymike00) [inline telegram/app/qml/AccountMessageMedia.qml]: we should change this icon, but idk which other one use to replace it...

**Comment by mymike00** [comment]
the icons here:
telegram-pr97-a
and these action for download (I put these photos in the old pr but I forgot to push the code...)
newtelegram03
newtelegram04

File: telegram/app/qml/AccountMessageMedia.qml (hunk 1 above)
Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: hmmmm let me see the design template...

**Comment by Flohack74** [comment]
By the way for the telephone number, can you change this from the one with the plus sign to "phone-uncategorized-symbolic" ? I think thats the right one ^^

File: telegram/app/qml/AccountMessageMedia.qml (hunk 1 above)
Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: Hmm for what is it used? ^^

Comment (mymike00) [reply telegram/app/qml/AccountMessageMedia.qml]: when a document is sent, I think, and while downloading a sticker
and it's quite horrible for the Suru theme
document

Comment (mymike00) [reply telegram/app/qml/AccountMessageMedia.qml]: maybe we can use this from suru mimetypes
x-office-document-symbolic svg
or this (the docviewer app without the lens, or if you prefer with the lens...)
doc-app svg
or else an icon from the new redesign form Sam Hewitt...

**Comment by mymike00** [comment]
whatever you want... I put that because it copies the phone number in the dialer and doesn't start the call immediately, but its the same...

File: telegram/app/qml/AccountMessageMedia.qml (hunk 1 above)
Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: Maybe better stock_document ^^ I agree that this is horrible

Comment (mymike00) [reply telegram/app/qml/AccountMessageMedia.qml]: Could this one be better?

stock_document svg
from: `suru/actions/scalable`

Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: Yeah lets go with this one ;)

Comment (mymike00) [reply telegram/app/qml/AccountMessageMedia.qml]: fixed in PR #111

Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: Nah this was comment about the last document icon you proposed, I never answered ;)

Comment (mymike00) [reply telegram/app/qml/AccountMessageMedia.qml]: what do you never answered? You approved my last icon so I changed it in a Pull Request I already had open...

Comment (Flohack74) [reply telegram/app/qml/AccountMessageMedia.qml]: Ok then its good ;)

**Pull request merged**